ccd-definition/CaseEventToFields/DefendantResponseLRspec.json

```diff
@@ -219,6 +219,17 @@
     "FieldShowCondition": "defenceRouteRequired = \"HAS_PAID_THE_AMOUNT_CLAIMED\"",
     "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/mid/track"
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
+    "CaseFieldID": "respondent1ClaimResponsePaymentAdmissionForSpec",
+    "FieldShowCondition": "specAoSRespondentDetails = \"DO_NOT_SHOW_IN_UI\" OR respondent1ClaimResponseTypeForSpec = \"DO_NOT_SHOW_IN_UI\"",
```

Comment:
Not a big thing, this will never evaluate is true so we dont need an OR operator here.
Just incase someone looks at this in future and thinks this should be shown somewhere

Reply:
Removed 
```
OR respondent1ClaimResponseTypeForSpec = \"DO_NOT_SHOW_IN_UI\"
```